src/Post.php

```diff
@@ -554,9 +553,9 @@ public function reading_time(): ?int
         }
 
         $locale = $this->get_locale();
-        $wpm = planet4_get_option('reading_time_wpm', Post\ReadingTimeCalculator::DEFAULT_WPM);
+        $wpm = Settings\ReadingTime::get_option();
         $rt = new Post\ReadingTimeCalculator($locale, $wpm);
-        $time = (int) max(1, round($rt->get_time($this->content) / 60));
+        $time = (int) max(1, round($rt->get_time($this->content() ?? '') / 60));
```

Comment:
Shouldn't we use `0` here instead of `''`?

Reply:
ah, no it's the post content, I want it to be a string even if content() returns null or fails - ReadingTimeCalculator::get_time() takes a string as arg

Reply:
Ah, right, I got confused with the parenthesis 🙈 makes sense!